tests/users/test_api.py

```diff
@@ -295,3 +292,25 @@ async def test_remove_permission(spawn_client, role, status, snapshot):
 
     assert resp.status == status
     assert await resp.json() == snapshot()
+
+
+async def test_first_user_view(spawn_client, mongo):
+    """
+    Checks response when first user exists and does not exist.
+    """
+
+    client = await spawn_client(authorize=True, administrator=True)
+
+    resp = await client.put(
+        "/users/first", {"handle": "fred", "password": "hello_world"}
+    )
+
+    assert resp.status == 409
+
+    await mongo.users.delete_one({"handle": "bob"})
+
+    resp = await client.put(
+        "/users/first", {"handle": "fred", "password": "hello_world"}
+    )
+
+    assert resp.status == 201
```

Comment:
Please include a snapshot of the response body for the successful case. I don't think we really use it on the FE but returning the newly created user document is part of the API defn